website/blog/2021-02-05-dbt-project-checklist.md

```diff
@@ -156,7 +156,7 @@ This post is the checklist I created to guide our internal work, and I’m shari
 
 **Useful links**
 
-*   [Version control](/docs/guides/best-practices/#version-control-your-dbt-project)
```

Comment:
Should this URL be redirecting to /guides/best-practices#version-control-your-dbt-project, or /guides/legacy/best-practices#version-control-your-dbt-project? I'm not sure what's correct here.

Reply:
Hey @john-rock I'm seeing this at `/guides/legacy/best-practices#version-control-your-dbt-project`